### ISSUE TITLE
Selecting and replacing value with "length === 1" leads to infinite loop in findChangedIndex

### ISSUE THREAD

**Issue by vadimgoncharov** [s-yadav/react-number-format#161]
You can reproduce it on demo page:
https://codepen.io/s-yadav/pen/bpKNMa

1. Focus first input
2. Write any character. For example: "1"
3. Select everything
4. Try to replace it with any character
5. Browser will be frozen

The problem is in `findChangedIndex`:
```
while (prevValue[i] === newValue[i]) { // <--- same length on both prevValue and newValue
    i++;
  } //check what has been changed from last
```

![screen capture on 2018-02-18 at 13-33-23](https://user-images.githubusercontent.com/357696/36350861-95a5b98e-14b0-11e8-9e06-5fa786a957a4.gif)

**Comment by s-yadav** [comment]
Fixed in v3.1.6 . Let me know if the issue still exists.

**Comment by vadimgoncharov** [comment]
Wow! Now everything works just fine :) Thanks!
